bambou/session.go

```diff
@@ -339,11 +374,13 @@ func (s *Session) DeleteEntity(object Identifiable) *Error {
 	}
 
 	request, err := http.NewRequest("DELETE", url, nil)
+
 	if err != nil {
-		return NewError(http.StatusBadRequest, err.Error())
+		return NewBambouError("", err.Error())
```

Comment:
Why are we not returning the bad request code as title? Not a big fan of empty titles